app/src/routes/+page.server.ts

```diff
@@ -7,18 +7,40 @@ import {
   externalPastEventsQuery,
 } from "$lib/server/sanity/queries";
 import { client } from "$lib/sanity/client";
+import { getAttendingEvent } from "$lib/server/supabase/queries";
 
 export const load: PageServerLoad = async ({ url, locals }) => {
   const auth = await locals.auth();
-
   const selectedCategory = url.searchParams.get("category")?.toLowerCase() || "";
 
-  const futureEvents: Event[] = await client.fetch(
-    auth?.user ? allFutureEventsQuery : externalFutureEventsQuery
-  );
-  const pastEvents: Event[] = await client.fetch(
-    auth?.user ? allPastEventsQuery : externalPastEventsQuery
-  );
+  if (auth?.user?.email) {
+    const events: Event[] = await client.fetch(allFutureEventsQuery);
+
+    const promises = events.map(async (event) => {
```

Comment:
Må først gjennom sanity og deretter i supabase for finne om brukeren deltar

Reply:
Det er litt wanky og ha en await i en map. Jeg kom på nå at jeg kan hente alle eventer til brukeren i supabase og hente en liste over document_id og sammenligne de. Da blir det bare en request kontra alle disse her 👍

Reply:
Fixed ✅